package.json

```diff
@@ -94,8 +94,7 @@
   "lint-staged": {
     "(src|__mocks__)/**/*.js": [
       "eslint --fix",
-      "prettier --write",
-      "yarn test --bail --findRelatedTests"
```

Comment:
Shame on you :D

Reply:
We can reactivate when we have time to write some proper tests